insanely_fast_whisper_rocm/core/integrations/diarization.py

```
@@ -0,0 +1,472 @@
+            "ffmpeg",
+            "-y",
+            "-i",
+            audio_path,
+            "-ar",
+            "16000",
+            "-ac",
+            "1",
+            tmp_path,
+        ]
+        try:
+            result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)